### DISCUSSION TITLE
Issue with Telegraf failing to scrape metrics even though in runnin state

### DISCUSSION THREAD

**Discussion by AnishAngadi1807** [telegraf/telegraf#1904] [General]
I have observed that the Telegraf agent fails to scrape metrics on many occasions on my Linux VM, even though it is in a running state. Consequently, we are receiving numerous false alerts.
Could you please explain why this happens? Has anyone else encountered a similar situation, and if so, what steps did they take to resolve the issue?
One step we took was to increase the scrape interval to 2 minutes. However, we noticed yesterday that Telegraf once again failed to scrape the metrics.

**Comment by MKRhere** [comment]
I think you have the [wrong Telegraf](https://www.influxdata.com/time-series-platform/telegraf/).

**Reply by AnishAngadi1807** [reply]
We have used this rpm telegraf-1.27.4-1.x86_64.rpm.
This is the doc we referred to download Telegraf
https://github.com/influxdata/telegraf/releases

**Reply by MKRhere** [reply]
The repo you're linking is not this repo. This is a different telegraf. Please take your issue to https://github.com/influxdata/telegraf/issues
